### PR TITLE
CSS Inputs

### PR DESCRIPTION
**Pull request by mikegioia**

### CODE REVIEW

File: dist/css/uikit.css [hunk 1]
```diff
@@ -1533,6 +1544,147 @@ label.with-checkbox input {
...
+}
+.form-check-input.input-sm + .form-check-icon + span {
+  font-size: 0.8125rem;
+}
+.form-check-input[type="radio"] + .form-check-icon {
+  border-radius: 50%;
+}
+.form-check-input[type="radio"]:checked + .form-check-icon:before {
+  content: "\f111";
+  left: 0.05rem;
```
Comment (jlappano) [inline dist/css/uikit.css]: This white circle element isn't quite centered, though this is inconsistent. Found in the "form type editor":
screen shot 2018-01-03 at 11 16 59 am

found in the "create a new template" modal:
screen shot 2018-01-03 at 10 50 45 am

the `left` style attribute is frustratingly off by .04rem

**Review by jlappano — COMMENTED** [review]
@mikegioia

**Comment by taliator** [comment]
@jlappano I can look at this on Friday, does that work?

**Comment by mikegioia** [comment]
@taliator Sorry Talia, that comment/request was for me! I'm gonna merge it all tomorrow.

**Comment by taliator** [comment]
👍

File: dist/css/uikit.css (hunk 1 above)
Comment (mikegioia) [reply dist/css/uikit.css]: Just pushed up a fix for this. I'm pretty confident in it, too 😜  I'll have to show you tomorrow

Everything looks good to go, too. Thanks Julie!

**Comment by mikegioia** [comment]
This is ready to merge. @jlappano Could you please mark it approved?

**Review by jlappano — APPROVED** [review]

**Pull request merged**